ee/clickhouse/queries/session_recordings/clickhouse_session_recording_list.py

```diff
@@ -131,6 +132,28 @@ class ClickhouseSessionRecordingList(ClickhouseEventQuery):
     LIMIT %(limit)s OFFSET %(offset)s
     """
 
+    def __init__(
```

Comment:
Q: Why is this change needed? This is unrelated to stickiness and creates a lot of noise in the code.